components/fonts/font_context.rs

```diff
@@ -600,12 +716,26 @@ impl FontContext {
     }
 }
 
+pub(crate) enum WebFontLoadInitiator {
+    Stylesheet(DocumentStyleSheet, WebFontLoadFinishedCallback),
+    Script(Box<dyn FnOnce(LowercaseFontFamilyName, Option<FontTemplate>) + Send>),
```

Comment:
Might be worth renaming `WebFontLoadFinishedCallback` to `StylesheetWebFontLoadFinishedCallback` and then making a type alias called `ScritWebFontFinishedCallback`.

```suggestion
    Stylesheet(DocumentStyleSheet, WebFontLoadFinishedCallback),
    Script(Box<dyn FnOnce(LowercaseFontFamilyName, Option<FontTemplate>) + Send>),
```